src/pudl/output/ferc1.py

```diff
@@ -1236,7 +1251,10 @@ def _aggregatable_dimension_tags(
         )
         .set_index(idx)
     )
-    table_dimensions_ferc1 = table_dimensions_ferc1.set_index(idx)
+    # don't include the corrections
```

Comment:
We're filtering these out since we always want the correction to have the same tags as the parent, and will add those tags in later, right?

Reply:
right! I'll add a more descriptive comment to confirm that.